server/run.sh

```diff
@@ -17,6 +17,11 @@ git clone $SOURCE_REPOSITORY_URL||jumpto sendstatusmail
 
 
 dirname=${SOURCE_REPOSITORY_URL##*/}
+if [[ "$dirname" == *.git ]]
```

Comment:
Tangential comment here. `dirname` is a shell command provided by `coreutils` package. Maybe we should use a different variable name?